docs/00_introduction/package-pipeline.md

```diff
@@ -0,0 +1,44 @@
+# The Garden Linux Package Pipeline
+
+Garden Linux is based on [Debian](https://www.debian.org), but it has a unique design for how packages are built.
+This pipeline might look overly complex on first sight, but is based on it's requirements and past experience.
+This document aims to explain the decisions and tradeoffs made.
+
+## The bigger picture
+
+Garden Linux release artifacts are disk images for various platforms such as kvm, cloud providers or bare metal hardware.
+To build those artifacts, Garden Linux makes use of an [APT repository](https://wiki.debian.org/DebianRepository).
+APT is the Debian package management tool.
+
+Unlike Debian, Garden Linux is focussed on containers.
+This means that for users of Garden Linux, the APT repo has much less direct importance.
+Typically, we don't expect our users to install packages from the APT repository.
+
+The APT repo exists because it is needed for building our release artifacts (the disk images).
+
+## Qualities of the Garden Linux package pipeline
+
+Based on previous experience, the design goals of our package pipeline are the following:
+
+- We want to ship a stable operating system with a modern software stack
+- We want the latest LTS linux kernel with all the Debian benefits as described in [Linux Kernel](./kernel.md)
+- We want to be able to reproduce release artifacts (disk images) and patch individual packages as needed
+- We require updated packages (rebuilt from source, with patches applied) to be absolutely binary compatible with existing releases
+  - This is a challenge because Garden Linux is based on Debian *Testing* where components such as libc or the go compiler might change in incompatible ways without notice
```

Comment:
> without notice
reads like this silently fails, but apt package management will explicitly complain about dependency issues.

But, yes, we cannot subscribe to a mailing list, consume an API or stuff like that..